torch/testing/_internal/common_methods_invocations.py

```diff
@@ -18477,11 +18480,18 @@ def sample_inputs_alias_copy(op_info, device, dtype, requires_grad, **kwargs):
            )),
     OpInfo('sort',
            dtypes=all_types_and(torch.bool, torch.float16, torch.bfloat16),
-           dtypesIfCUDA=all_types_and(torch.float16, torch.bfloat16),
+           dtypesIfCUDA=all_types_and(torch.bool, torch.float16, torch.bfloat16),
            sample_inputs_func=sample_inputs_sort,
            supports_forward_ad=True,
            supports_fwgrad_bwgrad=True,
            skips=(
+                DecorateInfo(
```

Comment:
Need to add this because the non-standard bools (uint8) will matter for the sort order (at least for the indices). Wondering if there are some better ideas

Reply:
I'm OK with this if the behavior is determined by e.g., cub and we do not care to put in a workaround